clipper_admin/clipper_manager.py

```diff
@@ -58,20 +58,32 @@
 EXTERNALLY_MANAGED_MODEL = "EXTERNAL"
 
 
+from numpy import *
+import scipy as sp
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+from pandas import *
+import pandas.rpy.common as com
```

Comment:
This module no longer exists (see the warning at the top of this page https://pandas.pydata.org/pandas-docs/stable/r_interface.html).